appveyor.yml

```diff
@@ -1,10 +1,13 @@
 version: v1.11.0.build{build}
 
+image:
+  - Visual Studio 2017
+
 install:
   - cinst -y nsis
 
 matrix:
-  fast_finish: true
+  fast_finish: false
```

Comment:
`true` will stop the build on first failure (making the `allow_failures:` lines contradictory)